data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/datatype/postgres/handler/NetworkAddressTypeHandler.java

```diff
@@ -0,0 +1,14 @@
+package org.opensearch.dataprepper.plugins.source.rds.datatype.postgres.handler;
+
+import org.opensearch.dataprepper.plugins.source.rds.datatype.postgres.PostgresDataType;
+import org.opensearch.dataprepper.plugins.source.rds.datatype.postgres.PostgresDataTypeHandler;
+
+public class NetworkAddressTypeHandler implements PostgresDataTypeHandler {
+    @Override
+    public Object handle(PostgresDataType columnType, String columnName, Object value) {
+        if (!columnType.isNetworkAddress()) {
+            throw new IllegalArgumentException("ColumnType is not Network Address: " + columnType);
+        }
+        return value.toString();
```

Comment:
I thought we are converting Json to a map so that it will be an object in OpenSearch documents. But on the other hand, users can configure parse_json processor to parse it if necessary. Is it consistent with how we handle json in MySQL data type mapping?

Reply:
I'm referring to the JsonTypeHandler. Somehow added at the wrong place.